design/tenant.go

```diff
@@ -170,6 +170,22 @@ var _ = a.Resource("tenants", func() {
 		a.Response(d.Unauthorized, JSONAPIErrors)
 	})
 
+	a.Action("delete", func() {
+		a.Security("jwt")
+		a.Routing(
+			a.GET("/:tenantID"),
```

Comment:
tenantID is an identity/user ID, right?

Reply:
yes, it's called `tenant` in this service